src/pages/laced.pug

```diff
@@ -0,0 +1,84 @@
+            detailText: "React, Ruby on Rails, Stripe, PayPal, UPS"
+          },
+          {
+            detailHeading: "Platform",
+            detailText: "Web"
+          }
+        ]
+      })
+
+  .section
+    .container
+      figure
+        img(src="/assets/images/laced/inline-1.webp" alt="The HEHA! iOS application" onerror="this.onerror=null; this.src='/assets/images/laced/inline-1.jpg'")
+
+      .grid.grid--two-thirds-reverse
+        .grid__item
+          +summary({
+            summaryHeading: "What is it?",
+            summaryText: "Laced allows sneaker enthusiasts to buy and sell collectible sneakers online, as well as track market values, and reserve items at certain prices."
+          })
+
+        .grid__item
+          h4 The challenge
+          p Laced are up against some huge competition from established sneaker trading platforms in the United States. Spotting an opportunity to own the European market, they needed a high quality automated service that could compete with the big players delivered quickly in order to maintain a competitive edge.
+
+          h4 Our approach
+          p Before Laced, European sneaker enthusiasts were having to trade their shoes on the more generalist platforms like eBay, Facebook, or one of a few poorly executed European sneaker trading platforms available.
+
+          p By using a combination of Ruby on Rails and React, we were able to quickly build Laced a high quality online trading platform with a seamless user interface for their customers. Both technologies also allow us to rapidly develop future features and service integrations, setting Laced up with a strong competitive edge.
+
+      figure
+        img(src="/assets/images/laced/inline-2.webp" alt="The HEHA! iOS application" onerror="this.onerror=null; this.src='/assets/images/laced/inline-2.jpg'")
+
+      .grid.grid--two-thirds
+        .grid__item
+          h4 Fully automated payments
+          p As a new name in the industry, loyalty and trust are of extra importance to Laced. To handle payments securely and robustly, we integrated with Stripe and PayPal. Our two integrations allow customers to make immediate purchases, or add credit to their account balance which they can use to reserve items at a certain price.
+          p Sellers can also withdraw the money they've made through Laced directly into their own bank accounts, with zero human involvement.
+
+          h4 Shipping and quality control
+          p One of the main hassles of trading physical items online is having to arrange shipping and quality control. We automated this by allowing sellers to print out pre-paid shipping labels through a UPS integration, and forcing all items to be shipped via Laced's headquarters, where they're quality inspected before being sent on to buyers. This way, users can ship products directly from their homes, buyers and sellers remain anonymous to one another, and all products have their quality guaranteed before delivery.
```

Comment:
Consider: We automated this `process`. I think it reads smoother, and for me, the word 'process' helps describe our own definition of automation, and what we mean by it.